<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgCameras</class>
 <widget class="QDialog" name="CQDlgCameras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Properties</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>131</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="title">
    <string>Main properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>241</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Perspect. angle [deg] / ortho. size [m]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqPerspectiveProjectionAngle">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>25</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqOrthographicProjectionSize">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>25</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>241</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Near/far clipping plane [m]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqNearClipping">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>45</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqFarClipping">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>45</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Tracked object</string>
    </property>
   </widget>
   <widget class="QComboBox" name="qqTrackedCombo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>85</y>
      <width>221</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>50</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqManipProxy">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>105</y>
      <width>221</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Parent as manipulation proxy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>45</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>25</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Render mode</string>
    </property>
   </widget>
   <widget class="QComboBox" name="qqRenderModeCombo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>65</y>
      <width>221</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>411</width>
     <height>90</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="title">
    <string>Manipulation mode permissions</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqAlongX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Along own X-axis</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqAlongY">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Along own Y-axis</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqAlongZ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Along own Z-axis</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqFullRotation">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>25</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Full rotation</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqTilting">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>45</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Tilting</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqShowFog">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>65</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Fog</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqLocalLights">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>65</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Local lights</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqAllowPicking">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>65</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Picking</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="title">
    <string>Visual properties</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="qqColorA">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>25</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Adjust color</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqSize">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>25</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Object size [m]</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqShowVolume">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Show view frustum</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqPerspectiveProjectionAngle</tabstop>
  <tabstop>qqOrthographicProjectionSize</tabstop>
  <tabstop>qqNearClipping</tabstop>
  <tabstop>qqFarClipping</tabstop>
  <tabstop>qqRenderModeCombo</tabstop>
  <tabstop>qqTrackedCombo</tabstop>
  <tabstop>qqManipProxy</tabstop>
  <tabstop>qqAlongX</tabstop>
  <tabstop>qqAlongY</tabstop>
  <tabstop>qqAlongZ</tabstop>
  <tabstop>qqFullRotation</tabstop>
  <tabstop>qqTilting</tabstop>
  <tabstop>qqShowFog</tabstop>
  <tabstop>qqLocalLights</tabstop>
  <tabstop>qqAllowPicking</tabstop>
  <tabstop>qqSize</tabstop>
  <tabstop>qqColorA</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
